packages/core/src/browser/shell/sidebar-menu-widget.tsx

```diff
@@ -20,9 +20,9 @@ import { ReactWidget } from '../widgets';
 import { ContextMenuRenderer } from '../context-menu-renderer';
 import { MenuPath } from '../../common/menu';
 
-export const SidebarBottomMenuWidgetFactory = Symbol('SidebarBottomMenuWidgetFactory');
+export const SidebarMenuWidgetFactory = Symbol('SidebarMenuWidgetFactory');
 
-export interface SidebarBottomMenu {
+export interface SidebarMenu {
```

Comment:
Renaming `SidebarBottomMenu` is a breaking change, and should be documented as such.